### PR TITLE
Use superbuild

### PR DESCRIPTION
**Pull request by nurmukhametov**
It uses LLVM version from os presets that is always actual, e.g., LLVM 17 at the moment.

### CODE REVIEW

**Comment by nurmukhametov** [comment]
It fixes https://github.com/compiler-explorer/compiler-explorer/issues/6347

File: build/build.sh [hunk 1]
```diff
@@ -48,26 +47,21 @@ if [[ "${REVISION}" == "${LAST_REVISION}" ]]; then
...
 fi
 
-export LLVM_HOME=${ROOT}/llvm
-export ISPC_HOME=${ROOT}/ispc
+ISPC_HOME=${ROOT}/ispc
 
 # Checkout
 git clone --depth 1 --single-branch -b "${BRANCH}" "${URL}" "${ISPC_HOME}"
 
-cd "${ISPC_HOME}"
```
Comment (mattgodbolt) [inline build/build.sh:57]: Without this where does the LLVM come from?

Comment (nurmukhametov) [reply build/build.sh:57]: LLVM comes from `LLVM_URL` and `LLVM_VERSION` defined here https://github.com/ispc/ispc/blob/main/superbuild/osPresets.json

**Review by mattgodbolt — APPROVED** [review]
Discussed on Discord: thanks for your patience!

**Pull request merged**